.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "chrome",
```

Comment:
I don't use chrome :p 
You should add this file to the .gitignore instead

Reply:
Oh lol. I wonder how they do in companies in this situation 🤔 Because sometimes it is useful to share some setups in the `launch.json`

But yeah I can ignore it here

Reply:
Add to the `.gitignore` in [chore: Add launch.json to .gitignore](https://github.com/pyronear/pyro-crowdsourcer-react/pull/5/commits/45dbc56f0989e77084a17bb24c47f661542e41e4)